orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/StageExecutionImpl.java

```diff
@@ -614,6 +614,16 @@ public <O> O mapTo(@Nullable String pointer, @Nonnull Class<O> type) {
     }
   }
 
+  @Override
+  public boolean isManualJudgmentType() {
+    return this.type.equals("manualJudgment");
+  }
+
+  @Override
+  public boolean withPropagateAuthentication() {
+    return Boolean.parseBoolean(context.get("propagateAuthenticationContext").toString());
```

Comment:
What if `context.get("propagateAuthenticationContext")` returns null?

Reply:
I added an extra validation to prevent convert a null toString. Thank you for the observation